src/model.rs

```diff
@@ -387,7 +392,7 @@ pub struct Group {
 	pub channel_id: ChannelId,
 	pub icon: Option<String>,
 	pub last_message_id: Option<MessageId>,
-	pub last_pin_timestamp: Option<String>,
+	pub last_pin_timestamp: Option<DateTime<FixedOffset>>,
```

Comment:
How does serde interpret this? Does it work with the RFC strings by default or does there need to be a wrapper?